modules/custom/openy_hf/openy_hf.module

```diff
@@ -51,6 +51,9 @@ function openy_hf_page_attachments(array &$attachments) {
 function openy_hf_preprocess_html(&$variables) {
   $route = \Drupal::service('current_route_match')->getRouteName();
   if ($route == 'openy_hf.header' || $route == 'openy_hf.footer') {
+    // Remove Admin menu in header and footer template.
+    unset($variables['page_top']['toolbar']);
```

Comment:
Let's wrap into `if` condition. We can't garantee that this element will be always available.
```php
if (isset($variables['page_top']['toolbar'])) {
  unset($variables['page_top']['toolbar']);
}
```